azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -309,6 +316,12 @@ public String getSlaOptionStr() {
     flowObj.put(FLOW_LOCK_ERROR_MESSAGE_PARAM, this.flowLockErrorMessage);
     flowObj.put(FLOW_DISPATCH_METHOD, getDispatchMethod().getNumVal());
 
+    if (this.getVersionSet() != null) {
```

Comment:
Can you please add a comment about version set? Please state when it will be null and when it wont be.

Reply:
Added a comment